src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -607,6 +611,47 @@ const VisitorSpaceSearchPage: FC = () => {
 		);
 	};
 
+	const renderTempAccessLabel = () => {
+		return (
+			<div className="p-visitor-space__temp-access-container">
+				<Icon name={IconNamesLight.Clock} />
+				<span className="p-visitor-space__temp-access-label">
+					{tText(
+						'modules/visitor-space/components/visitor-space-search-page/visitor-space-search-page___tijdelijke-toegang'
+					)}
+					{dropdownOptions
```

Comment:
Add a hardcoded space between the translation and the links:

![image](https://user-images.githubusercontent.com/1710840/226559573-c9bb55d2-d45d-4659-8ff0-a1ddd7b2224c.png)


```suggestion
					{tText(
						'modules/visitor-space/components/visitor-space-search-page/visitor-space-search-page___tijdelijke-toegang'
					)}
					{' '}
					{dropdownOptions
```